models/sipnet/R/write.configs.SIPNET.R

```diff
@@ -437,7 +447,26 @@ write.config.SIPNET <- function(defaults, trait.values, settings, run.id, inputs
     if ("leafGrowth" %in% pft.names) {
       param[which(param[, 1] == "leafGrowth"), 2] <- pft.traits[which(pft.names == "leafGrowth")]
     }
-  }  ## end loop over PFTS
+
+    #update LeafOnday and LeafOffDay
+    if (update_phenology){
+     leaf_pheno_outdir <- settings$model$leaf_phenology$outdir  ## read from settings
```

Comment:
1. the leaf phenology data is an *INPUT* and thus should be referenced like all other inputs, e.g. settings$run$inputs$leaf_phenology. 
2. As such leaf_phenology should be registered with BETY as a new optional input to SIPNET.
3. The SIPNET run is not writing anything new to leaf_phenology$outdir and thus it shouldn't be called `outdir` but should use `id`, `path`, or `source` like all other inputs, depending on whether you're specifying a database ID, a file path, or the name of the input source that is then processed within do.conversions.
4. It's worth noting that prescribed phenology is not a SIPNET-specific feature, but something that could be valuable to any model, which is an additional argument for moving it to inputs instead of model